code/datums/traits/negative.dm

```diff
@@ -138,7 +138,7 @@
 			if("Chaplain")
 				heirloom_type = pick(/obj/item/toy/windupToolbox, /obj/item/reagent_containers/food/drinks/bottle/holywater)
 			if("Assistant")
-				heirloom_type = pick(/obj/item/storage/toolbox/mechanical/old/heirloom, /obj/item/clothing/gloves/cut/heirloom)
+				heirloom_type = pick(/obj/item/heirloomtoolbox)//, /obj/item/clothing/gloves/cut/heirloom)
```

Comment:
```suggestion
				heirloom_type = /obj/item/heirloomtoolbox
```

Reply:
Why remove the gloves though?

Reply:
Fuck, the glove removal was totally by accident.  it's commented out because I intended to put it back after testing lmao.